sdk/src/main/java/com/meniga/sdk/helpers/Result.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2017-2018 Meniga Iceland Inc.
+ */
+package com.meniga.sdk.helpers
+
+import com.meniga.sdk.providers.tasks.Task
+
+interface Result<T> {
```

Comment:
Do you think it makes sense to keep this interface in the sdk still? It was meant so that clients could implement different call handling solutions but right now it seems more of an unnecessary indirection for Tasks.

Reply:
You are right about that @agustk. I just don't want to break the API for the client yet (we return `Result<T>` everywhere). I guess we can have a separate PR for that.